diff --git a/scripts/bootstrap/compile.sh b/scripts/bootstrap/compile.sh
index 750ff07..c0f91a5 100755
--- a/scripts/bootstrap/compile.sh
+++ b/scripts/bootstrap/compile.sh
@@ -144,7 +144,7 @@ function java_compilation() {
   # Useful if your system chooses too small of a max heap for javac.
   # We intentionally rely on shell word splitting to allow multiple
   # additional arguments to be passed to javac.
-  run "${JAVAC}" -classpath "${classpath}" -sourcepath "${sourcepath}" \
+  run "${JAVAC}" -verbose -classpath "${classpath}" -sourcepath "${sourcepath}" \
       -d "${output}/classes" -source "$JAVA_VERSION" -target "$JAVA_VERSION" \
       -encoding UTF-8 ${BAZEL_JAVAC_OPTS} "@${paramfile}"
 
@@ -170,7 +170,7 @@ function create_deploy_jar() {
 
   log "Creating $name.jar..."
   echo "Main-Class: $mainClass" > $output/MANIFEST.MF
-  run "$JAR" cmf $output/MANIFEST.MF $output/$name.jar $packages "$@"
+  run "$JAR" -verbose cmf $output/MANIFEST.MF $output/$name.jar $packages "$@"
 }
 
 HOW_TO_BOOTSTRAP='
