Description: Fix bashisms
 File eliminated upstream in tag 6.0.0-pre.20220310.1
Author: Olek Wojnar <olek@debian.org>
Forwarded: not-needed
Last-Update: 2022-10-17

--- a/scripts/docs/jekyll_build.sh.tpl
+++ b/scripts/docs/jekyll_build.sh.tpl
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 HOST="${HOST-localhost}"
 PORT="${PORT-12345}"
 RUNFILES=$(cd ${JAVA_RUNFILES-$0.runfiles}/%{workspace_name} && pwd -P)
